README.md

```diff
@@ -23,7 +23,7 @@ Enhanced WebView component for Android that works as intended out of the box
 
      ```gradle
      dependencies {
-         compile 'com.github.delight-im:Android-AdvancedWebView:v3.0.0'
+         implementation("com.github.delight-im:Android-AdvancedWebView:v3.2.0")
```

Comment:
Nice bumping the version and using implementation. I don't think we need the braces though.

Reply:
No braces are required. I tried your input in kotlin DSL but the gradle failed. Above is correct

Reply:
The readme does not suggest that Kotlin DSL is used. I'd say either mention it, or use the more conventional syntax.

Reply:
The above syntax is common for both gradle & kotlin DSL